src/js/analyses/components/Create/WorkflowSelector.js

```diff
@@ -17,24 +92,45 @@ export const getCompatibleWorkflows = (dataType, hasHmm) => {
 };
 
 export const WorkflowSelector = ({ dataType, hasError, hasHmm, workflows, onSelect }) => {
-    const workflowItems = map(getCompatibleWorkflows(dataType, hasHmm), workflow => (
-        <MultiSelectorItem key={workflow} name={workflow} value={workflow}>
-            {getWorkflowDisplayName(workflow)}
-        </MultiSelectorItem>
-    ));
+    const clearSelected = object => {
+        onSelect(workflows.filter(workflow => workflow !== object));
+    };
+
+    const workflowItems = map(getCompatibleWorkflows(dataType, hasHmm), workflow =>
+        !workflows.includes(workflow) ? (
+            <SelectorItem key={workflow} name={workflow} value={workflow} requireCheckbox={false} id={workflow}>
+                {getWorkflowDisplayName(workflow)}
+            </SelectorItem>
+        ) : null
+    );
```

Comment:
workflows has been renamed to selectedWorkflows. Negation still must be present since the rendering is occuring if the workflow is not in fact selected. The other change has been implemented.